### PR TITLE
Add confetti to end page

### PR DESCRIPTION
**Pull request by jacknugent**
https://user-images.githubusercontent.com/31449177/133005941-f54b578e-483f-4e27-aa78-02477af0220a.mov

### CODE REVIEW

**Comment by kevincianfarini** [comment]
amazing!

File: frontend/src/App.tsx [hunk 1]
```diff
@@ -10,7 +10,7 @@ import PickYourIssue from './Pages/PickYourIssue/PickYourIssue';
 import SendAnEmail from './Pages/SendAnEmail/SendAnEmail';
 import MakeAPhoneCall from './Pages/MakeAPhoneCall';
 import PostOnSocial from './Pages/PostOnSocial';
-import AllDone from './Pages/AllDone';
+import AllDone from './Pages/AllDone/AllDone';
```
Comment (kevincianfarini) [inline frontend/src/App.tsx:13]: What's going on here? This seems like a weird import.

**Comment by kevincianfarini** [comment]
Approving and merging for now. Comment is a nit.

**Review by kevincianfarini — APPROVED** [review]

File: frontend/src/App.tsx (hunk 1 above)
Comment (jacknugent) [reply frontend/src/App.tsx:13]: It's a component inside of a folder. I moved the component so I could add a css file associated with the component, better organization that way

**Pull request merged**